paparazzi/src/main/java/app/cash/paparazzi/accessibility/AccessibilityOverlayDrawable.kt

```diff
@@ -0,0 +1,41 @@
+package app.cash.paparazzi.accessibility
+
+import android.graphics.Canvas
+import android.graphics.ColorFilter
+import android.graphics.Paint
+import android.graphics.drawable.Drawable
+import app.cash.paparazzi.accessibility.RenderSettings.toColorInt
+
+internal class AccessibilityOverlayDrawable : Drawable() {
```

Comment:
Ended up making this a foreground drawable instead of a view so it can be drawn overtop WindowManagerImpl

Reply:
cool